assets/app/views/_pods.html

```diff
@@ -1,11 +1,12 @@
 <!--
-  Expects the following variables:
-  pods - a hash of pod objects by id
+Expects the following variables:
+pods - a hash of pod objects by id
 -->
 <div class="component-block pod-container">
-  <div class="animate-repeat pod pod-{{pod.status.phase.toLowerCase()}} osc-object"  ng-repeat="pod in pods track by (pod | uid)" kind="Pod" resource="pod">
+  <div class="animate-repeat pod pod-{{pod.status.phase.toLowerCase()}} osc-object"  ng-repeat="pod in pods track by (pod | uid)" kind="Pod" resource="pod" ng-class="{'pod-warning': (pod | isTroubledPod)}">
```

Comment:
should combine `pod-{{pod.status.phase.toLowerCase()}` with the troubled pod check... we probably shouldn't set both pod-running and pod-warning